ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/event-handlers.j2

```diff
@@ -12,27 +12,30 @@ event-handler {{ handler.name }}
 {%             if handler.trigger is arista.avd.defined("on-maintenance")
                   and handler.trigger_on_maintenance.operation is arista.avd.defined
                   and handler.trigger_on_maintenance.action is arista.avd.defined %}
-{%                 set trigger_on_manitenance_cli =  "trigger " ~ handler.trigger ~ " " ~ handler.trigger_on_maintenance.operation %}
-{%                 if handler.trigger_on_maintenance.bgp_peer is arista.avd.defined %}
-{%                     set trigger_on_manitenance_cli = trigger_on_manitenance_cli ~ " bgp " ~ handler.trigger_on_maintenance.bgp_peer %}
-{%                     if handler.trigger_on_maintenance.vrf is arista.avd.defined %}
-{%                         set trigger_on_manitenance_cli = trigger_on_manitenance_cli ~ " vrf " ~ handler.trigger_on_maintenance.vrf %}
+{%                 set on_maintenance_cli = [] %}
+{%                 for trigger in ["bgp_peer", "interface", "unit"] %}
```

Comment:
As any one trigger can be used in config so can we just have three if elif conditions for trigger then common conditions on action and stage. Right now its seems we can have all three trigger in config which is not valid.

Reply:
Fixed